index.html

```diff
@@ -154,8 +202,9 @@ <h1 class="intro-title1">PyDelhi<br />conference</h1>
                 <!-- <div id="countdown" class="row">March 5, 2016 @ New Delhi</div> -->
                 <!-- /countDown -->
                 <!-- <p class="countdown-title2"> -->
-
+                <a href="#section-schedule" class="btn"><i class="fa fa-play"></i>&nbsp;Register Yourself</a>  
                 <a href="https://www.papercall.io/pydelhiconf2023" class="btn"><i class="fa fa-play"></i>&nbsp;Submit Proposal</a>
+                <a href="data/SponsorshipProspectus2023.pdf" class="btn"><i class="fa fa-play"></i>&nbsp;Sponsor Prospectus</a>
```

Comment:
Please add target="_blank" so that these anchor tags open in new tab for both proposal link and sponsorship prospectus.

Reply:
Done